otter/test/convergence/test_service.py

```diff
@@ -848,12 +856,13 @@ def get_seq(self, with_cache=True):
         ]
 
     def _invoke(self, plan=None):
-        kwargs = {'plan_launch_server': plan} if plan is not None else {}
+        kwargs = {'plan': plan} if plan is not None else {}
+        executor = LaunchServerExecutor(gather=intent_func("gacd"), **kwargs)
```

Comment:
and if you go with option 2 above this would be something like `attr.assoc(launch_server_executor, gather=my_gather, ...)`